packages/core/strapi/tests/api/relations.test.api.js

```diff
@@ -138,6 +138,7 @@ const createShop = async ({
   anyToManyRel = [{ id: id1 }, { id: id2 }, { id: id3 }],
   data = {},
   populate,
+  strictConnect = true,
```

Comment:
Why do you put it explicitely to true in the test if it's already the default value? Should there be a test to verify that the default value is true?

Reply:
haven't thought of that! I will not set any default value to test that :)